### ISSUE TITLE
[enhancement]: Face Detection & Facial Expression Change

### ISSUE THREAD

**Issue by destromestro** [invoke-ai/InvokeAI#7228]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### What should this feature add?

I want to suggest a feature that could be extremely useful for all the INVOKE users. Is there a way to integrate a facial detection model and a facial expression editor in to invoke where we can detect faces and then control the expressions within the canvas without loosing the already generated facial features? That would be amazing.
like this on in Comfy UI - https://github.com/PowerHouseMan/ComfyUI-AdvancedLivePortrait

### Alternatives

Currently, I am going through a long route to get this done and it's exhausting. 

I generate characters using INVOKE. Then somtimes I want to keep the facial features but only change the facial expression. Inpaint allwyas changes the facial features and yes some control can be applied with Controlnet. But it's never as good as the exsiting libraries like AdvanceLivePortrait or Facepoke.

So from INVOKE canvas, I take a screenshot of the face I want to change, run it through Comfy ui with the AdvanceLivePortrait  nodes by PowerHouseMan or run through Facepoke, manually change the values and once satisfied, download the face with the new expression, then import thatto invoke, place it above the original face then try to blend the new one in with the rest of the image. It takes a lot of time and effort. 

Given that INVOKE is at the on the top of the GEN AI Art Creation platforms (hands down! there is nothing like it out there at the moment), it would be amazing to have a feature like this inbuilt wich will give immense conrol to the generated character facial expressions and will add a lot of value for INVOKE. Imagine the possibilities for comic artists and such.

**Comment by MexicoMLX** [comment]
Yes, that's exactly what we need. That would be just great. If anyone has any ideas on how to implement this in Invoke, please contact me.
